hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1588,6 +1588,11 @@ public Builder withWriteMetaKeyPrefixes(String writeMetaKeyPrefixes) {
       return this;
     }
 
+    public Builder withPopulateMetaFields(boolean populateMetaFields) {
```

Comment:
Can you please clarify me something. if we wish to store the property in hoodie.properties and is part of HoodieTableConfig, then we should set the config via HoodieWriteConfigBuilder.withHoodieTableConfig(new HoodieTableConfigBuilder().withPopulate ... sort of? and exposing setter here in HoodieWriteConfig is not the right way to go about?
It was easier for me in tests to set this param. but wanted to know whats the right way to go about.